app/src/test/java/com/arctouch/codechallenge/HomeControllerUnitTest.java

```diff
@@ -0,0 +1,54 @@
+package com.arctouch.codechallenge;
+
+import com.arctouch.codechallenge.controller.HomeController;
+import com.arctouch.codechallenge.data.Cache;
+import com.arctouch.codechallenge.model.Genre;
+import com.arctouch.codechallenge.model.Movie;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(Cache.class)
+public class HomeControllerUnitTest {
+
+    @Mock
+    HomeController homeController;
+
+    @Before
+    public void prepareForTest() {
+        homeController = new HomeController();
```

Comment:
You annotated your homeController with @Mock and it's been created here. Any special reason?

Reply:
Actually it shouldn't have that annotation as it is the object I am unit testing, when I developed I put it to try out a stub for a static method but I ended up using PowerMock to mock that static method. It is a fix to do :)